lib/node_modules/@stdlib/math/base/special/normhermite-polynomial/examples/index.js

```diff
@@ -18,6 +18,22 @@
 
 'use strict';
 
+var randu = require( '@stdlib/random/base/randu');
```

Comment:
Missing a space before the closing parenthesis. In short, we prefer balanced spacing.